src/components/Navbar/Navbar.jsx

```diff
@@ -38,18 +46,31 @@ const Navbar = () => {
                         </Link>
                     </li>
                     <li className="nav-main-li">
-                        <Link to="/login" className="nav-main-item nav-btn-login">
-                            Login
-                        </Link>
+                        {!auth ? (
+                            <Link to="/login" className="nav-main-item nav-btn-login">
+                                Login
+                            </Link>
+                        ) : (
+                            <div
+                                className="nav-main-item nav-btn-login"
+                                onClick={() => signOutHandler()}
+                            >
+                                Logout
+                            </div>
+                        )}
                     </li>
                     <li className="nav-main-li">
-                        <Link to="/products" className="nav-main-item">
-                            <i className="fa-solid fa-cart-shopping"></i>
+                        <Link to="/cart" className="nav-main-item badge">
+                            <i className="fa-solid fa-cart-shopping nav-right-icn"></i>
+                            {itemsInCart > 0 && <span className="badge-info">{itemsInCart}</span>}
                         </Link>
                     </li>
                     <li className="nav-main-li">
-                        <Link to="/products" className="nav-main-item">
-                            <i className="fa-regular fa-heart"></i>
+                        <Link to="/wishlist" className="nav-main-item badge">
+                            <i className="fa-regular fa-heart nav-right-icn"></i>
+                            {itemsInWishlist > 0 && (
+                                <span className="badge-info">{itemsInWishlist}</span>
+                            )}
```

Comment:
The wishlist icon in the navbar shows the badge even when the user is logged out. You can fix that.